test/coordtest/linearizability.ct

```diff
@@ -0,0 +1,107 @@
+inc-timestamp
+10
+----
+
+create-file name=s1
+1
+----
+
+create-file name=s2
+2
+----
+
+sql
+CREATE MATERIALIZED SOURCE s1 (a)
+FROM FILE '<TEMP>/s1'
+WITH (tail = true)
+FORMAT TEXT;
+----
+CreatedSource {
+    existed: false,
+}
+
+sql
+CREATE MATERIALIZED SOURCE s2 (a)
+FROM FILE '<TEMP>/s2'
+WITH (tail = true)
+FORMAT TEXT;
+----
+CreatedSource {
+    existed: false,
+}
+
+sql
+CREATE MATERIALIZED VIEW v1 AS SELECT count(*) AS c FROM s1;
+----
+CreatedView {
+    existed: false,
+}
+
+sql
+CREATE MATERIALIZED VIEW v2 AS SELECT count(*) AS c FROM s2;
+----
+CreatedView {
+    existed: false,
+}
+
+wait-sql
+SELECT
```

Comment:
just to double check my understanding -- this is expected to return true, true right?

Reply:
yes, `wait-sql` loops until all datums returned are true.